src/main/java/seedu/utility/StonksGraph.java

```diff
@@ -285,5 +286,26 @@ private void drawReport(FinancialTracker finances) {
         }
     }
 
-
+    private double determineBarValue(double max) {
+        double barValue;
+        if (max < 1) {
```

Comment:
might want to refactor this part by replacing the conditionals with a method to make the code more readable